scripts/datastore.js

```diff
@@ -0,0 +1,36 @@
+// Imports the Google Cloud client library
+const Datastore = require('@google-cloud/datastore');
+
+const projectId = 'cloud-ad-step-2020';
+
+// Creates a client
+const datastore = new Datastore({
+  projectId: projectId,
+});
+
+// The kind for the new entity
+const kind = 'User';
+// The name for the new entity
+const name = 'sampleUser1';
+//The email for the new entity
+const email = 'user@gmail.com'
+// The Cloud Datastore key for the new entity
+const userKey = datastore.key([kind, name, email]);
```

Comment:
based on [the retrieving records example here](https://googleapis.dev/nodejs/datastore/latest/Datastore.html#Datastore-examples), i think our key can just have the kind and either a string name or id value, and the email can be a part of the data